README
NAME
    Regexp::Common::AT::NICAT - Defines patterns for nic.at person handles

VERSION
    version 1.100860

SYNOPSIS
        use Regexp::Common 'AT::NICAT';

        my $value = '...';
        if ($value =~ /$RE{at}{nicathandle}/) {
           #  ...
        }

DESCRIPTION
    This module defines patterns for nic.at (Austrian domain registry)
    person handles to be used with Regexp::Common.

    Please consult the manual of Regexp::Common for a general description of
    the works of this interface. Do not use this module directly, but load
    it via *Regexp::Common*.

PATTERNS
    At the moment there is only one pattern, but more patterns for .at
    domain names will follow - they have some restrictions on length and
    characters to be used.

    $RE{at}{nicathandle}
        Provides a regex to match nic.at person handles.

INSTALLATION
    See perlmodinstall for information and options on installing Perl
    modules.

BUGS AND LIMITATIONS
    No bugs have been reported.

    Please report any bugs or feature requests through the web interface at
    <http://rt.cpan.org/Public/Dist/Display.html?Name=Regexp-Common-AT-NICAT
    >.

AVAILABILITY
    The latest version of this module is available from the Comprehensive
    Perl Archive Network (CPAN). Visit <http://www.perl.com/CPAN/> to find a
    CPAN site near you, or see
    <http://search.cpan.org/dist/Regexp-Common-AT-NICAT/>.

    The development version lives at
    <http://github.com/hanekomu/Regexp-Common-AT-NICAT/>. Instead of sending
    patches, please fork this project using the standard git and github
    infrastructure.

AUTHOR
      Marcel Gruenauer <marcel@cpan.org>

COPYRIGHT AND LICENSE
    This software is copyright (c) 2007 by Marcel Gruenauer.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.



Changes
Revision history for Perl extension Regexp-Common-AT-NICAT

1.100860  2010-03-27 15:27:03 Europe/Vienna
    - converted the distribution to Dist::Zilla-style

0.03  Thu Jul 25 15:27:43 CEST 2008 (Marcel Gruenauer <marcel@cpan.org>)
    - Changed dist style and Changes back to standard. People didn't like it -
      the nail that sticks out gets hammered down.
    - Added standard test files; this will also help with CPANTS.

0.02  Thu, 29 May 2008 12:06:46 -0000 (Marcel Gruenauer <marcel@cpan.org>)
    - set the version to 0.02
    - .shipit: added Twitter step
    - Makefile.PL: added auto_install() and process_templates()
    - lib/*: converted to template
    - added documentation; code style change
    - added t/perlcriticrc
    - updated MANIFEST
    - .shipit: fixed svk.tagpattern

0.01  Fri, 09 Nov 2007 11:24:49 -0000 (Marcel Gruenauer <marcel@cpan.org>)
    - original version


lib/Regexp/Common/AT/NICAT.pm
use 5.008;
use strict;
use warnings;

package Regexp::Common::AT::NICAT;
our $VERSION = '1.100860';
# ABSTRACT: Defines patterns for nic.at person handles
use Regexp::Common 'pattern';
pattern
  name   => [qw(at nicathandle)],
  create => '[A-Z]{2,4}\d+-NICAT';
1;


__END__
=pod

=for stopwords nic.at

=head1 NAME

Regexp::Common::AT::NICAT - Defines patterns for nic.at person handles

=head1 VERSION

version 1.100860

=head1 SYNOPSIS

    use Regexp::Common 'AT::NICAT';

    my $value = '...';
    if ($value =~ /$RE{at}{nicathandle}/) {
       #  ...
    }

=head1 DESCRIPTION

This module defines patterns for nic.at (Austrian domain registry) person
handles to be used with L<Regexp::Common>.

Please consult the manual of L<Regexp::Common> for a general description of
the works of this interface. Do not use this module directly, but load it via
I<Regexp::Common>.

=head1 PATTERNS

At the moment there is only one pattern, but more patterns for .at domain
names will follow - they have some restrictions on length and characters to be
used.

=over 4

=item $RE{at}{nicathandle}

Provides a regex to match nic.at person handles.

=back

=head1 INSTALLATION

See perlmodinstall for information and options on installing Perl modules.

=head1 BUGS AND LIMITATIONS

No bugs have been reported.

Please report any bugs or feature requests through the web interface at
L<http://rt.cpan.org/Public/Dist/Display.html?Name=Regexp-Common-AT-NICAT>.

=head1 AVAILABILITY

The latest version of this module is available from the Comprehensive Perl
Archive Network (CPAN). Visit L<http://www.perl.com/CPAN/> to find a CPAN
site near you, or see
L<http://search.cpan.org/dist/Regexp-Common-AT-NICAT/>.

The development version lives at
L<http://github.com/hanekomu/Regexp-Common-AT-NICAT/>.
Instead of sending patches, please fork this project using the standard git
and github infrastructure.

=head1 AUTHOR

  Marcel Gruenauer <marcel@cpan.org>

=head1 COPYRIGHT AND LICENSE

This software is copyright (c) 2007 by Marcel Gruenauer.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.

=cut



Makefile.PL

use strict;
use warnings;

 BEGIN { require 5.008; } 

use ExtUtils::MakeMaker 6.11;



my %WriteMakefileArgs = (
                       'test' => {
                                   'TESTS' => 't/*.t'
                                 },
                       'NAME' => 'Regexp::Common::AT::NICAT',
                       'DISTNAME' => 'Regexp-Common-AT-NICAT',
                       'CONFIGURE_REQUIRES' => {
                                                 'ExtUtils::MakeMaker' => '6.11'
                                               },
                       'AUTHOR' => 'Marcel Gruenauer <marcel@cpan.org>',
                       'BUILD_REQUIRES' => {},
                       'ABSTRACT' => 'Defines patterns for nic.at person handles',
                       'EXE_FILES' => [],
                       'VERSION' => '1.100860',
                       'PREREQ_PM' => {
                                        'Test::More' => '0.88',
                                        'Scalar::Util' => '0',
                                        'Regexp::Common' => '0',
                                        'English' => '0',
                                        'File::Find' => '0',
                                        'File::Temp' => '0',
                                        'Carp' => '0'
                                      },
                       'LICENSE' => 'perl'
                     );


delete $WriteMakefileArgs{LICENSE}
  unless eval { ExtUtils::MakeMaker->VERSION(6.31) };

WriteMakefile(%WriteMakefileArgs);



